src/couch/test/eunit/couch_js_tests.erl

```diff
@@ -344,6 +353,7 @@ should_bump_timing_and_call_stats(_) ->
 
 %% erlfmt-ignore
 should_exit_on_oom(_) ->
+    ok = meck:expect(config, get_integer, fun("couchdb", "os_process_timeout", _) -> 15000 end),
```

Comment:
If we have a couch app started might as well just use the `config:set(..., Persist = false)`

Reply:
Ah, right, this is more direct!